test/unittest/Database/DatabaseTests.cpp

```diff
@@ -3425,6 +3439,7 @@ TEST_F(database_tests, select_rtps_packets_lost)
 TEST_F(database_tests, select_rtps_bytes_lost)
 {
     data_output.clear();
+    db.select(DataKind::RTPS_BYTES_LOST, participant_id, writer_locator->id, src_ts, end_ts);
```

Comment:
This line is repeated just below asserting that it does not throw. Please, remove.

```suggestion
```